__tests__/app.test.js

```diff
@@ -76,4 +77,45 @@ describe('GET /api/articles/:article_id', () => {
             expect(body.msg).toBe('Not Found')
         })
     });
+});
+describe('GET /api/articles', () => {
+    it('GET: 200 - Should respond an array of article objects containing the correct keys and value types', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+            console.log(body)
+            expect(Array.isArray(body.articles)).toBe(true);
+            expect(body.articles.length).toBeGreaterThan(0);
+            body.articles.forEach((article) => {
+                expect(article).not.toHaveProperty('body')
+                expect(article).toHaveProperty('author' && 'title' && 'article_id' && 'topic' && 'created_at' && 'votes' && 'article_img_url' && 'comment_count')
+                expect(typeof article.author).toBe('string')
+                expect(typeof article.title).toBe('string')
+                expect(typeof article.article_id).toBe('number')
+                expect(typeof article.topic).toBe('string')
+                expect(new Date(article.created_at).toString()).not.toBe('Invalid Date')
+                expect(typeof article.article_img_url).toBe('string')
+                expect(typeof article.comment_count).toBe('string')
```

Comment:
Do we really want this to be a string 👀 It is returned from SQL as a string , but ideally a count should be a numeric data type 😄 How can you make this happen? (You'll be really glad you did this when it comes to front-end review week ! )